dffml/df/memory.py

```diff
@@ -1072,6 +1085,9 @@ def __init__(
     ) -> None:
         super().__init__(config, parent)
         self._stack = None
+        # Maps instance_name to OrchestratorContext
+        self.subflows = {}
+        # Maps instance_name to List[Input]
```

Comment:
```suggestion
```

Reply:
I think this comment was associated with the `dict` we moved into the other function.